Source/WebCore/page/scrolling/ScrollingTree.cpp

```diff
@@ -953,13 +953,13 @@ bool ScrollingTree::willWheelEventStartSwipeGesture(const PlatformWheelEvent& wh
 
     Locker locker { m_swipeStateLock };
 
-    if (wheelEvent.deltaX() > 0 && m_swipeState.mainFramePinnedState.left() && !m_swipeState.clientAllowedRubberBandableEdges.left())
+    if (wheelEvent.deltaX() > 0 && m_swipeState.mainFramePinnedState.left() && m_swipeState.clientAllowedRubberBandableEdges.left() == ScrollBounceBehavior::Never)
```

Comment:
if it's "based on size" and the size means it's not scrollable, should that allow swipe?